web_src/css/base.css

```diff
@@ -1073,9 +1068,7 @@ a.label,
 }
 
 .ui.cards a.card:hover,
-.ui.link.cards .card:not(.icon):hover,
-a.ui.card:hover,
-.ui.link.card:hover {
+a.ui.card:hover {
```

Comment:
Isn't `a.ui.card:hover` already contained in `a.card:hover`?

Reply:
`.ui.cards a.card:hover` has higher priority , can be use to overwrite other `.ui.cards a` styles.